content/SCALE/SCALETutorials/DataProtection/RsyncTasksSCALE.md

```diff
@@ -16,166 +16,157 @@ keywords:
-
-### Creating an SSH Mode Rsync Task
+You can use the SSS connection created in [Setting Up an SSH Connection](#setting-up-an-ssh-connection) or create a new connection while configuring the rsync task.
 
 1. Go to **Data Protection** and click **Add** on the **Rsync Tasks** widget to open the **Add Rsync Task** screen.
 
-{{< trueimage src="/images/SCALE/DataProtection/AddRsyncTaskSourceRemoteSSH.png" alt="Add Rsync Task - SSH Mode" id="Add Rsync Task - SSH Mode" >}}
+   {{< trueimage src="/images/SCALE/DataProtection/AddRsyncTaskSourceRemoteSSH.png" alt="Add Rsync Task - SSH Mode" id="Add Rsync Task - SSH Mode" >}}
 
 2. Enter or browse to the dataset or folder to sync with the remote server.
-    Use the <span class="material-icons">arrow_right</span> to the left of the **/mnt** folder and each folder listed on the tree to expand and browse through, then click on the name to populate the path field.
+   Use the <span class="material-icons">arrow_right</span> to the left of the **/mnt** folder and each folder listed on the tree to expand and browse through, then click on the name to populate the path field.
 
     {{< include file="/static/includes/FileExplorerFolderIcons.md" >}}
 
-3. Select a **User** account to perform the rsync task.
-    The user must have permissions to run an rsync on the remote server and read/write permission for the local dataset.
+3. Select the administration user for the local host TrueNAS system from the **User** dropdown. This is the user account to perform the rsync task.
+   The user must have read/write permissions for the local dataset.
 
 4. Set the **Direction** for the rsync task.
-    Select **Pull** to copy from the remote server to TrueNAS or **Push** to copy to the remote server.
-
-5. Select **SSH** as the connection mode from the **Rsync Mode** dropdown.
-    The SSH settings fields show.
+   Select **Pull** to copy from the remote server to TrueNAS or **Push** to copy to the remote server.
 
+5. Select **SSH** as the connection mode from the **Rsync Mode** dropdown to use an SSH connection. The settings fields show.
+   
+   Set this to **Module** if syncing with a non-TrueNAS remote system. See [Setting Up an Rsync Task Using Module Mode](#setting-up-an-rsync-task-using-module-modek) for more information.
+  
 6. Choose a connection method from the **Connect using** dropdown list.
 
-    If selecting **SSH private key stored in user's home directory**, enter the IP address or hostname of the remote system in **Remote Host**.
-        Use the format *username@remote_host* if the username differs on the remote host.
-        Enter the SSH port number for the remote system in **Remote SSH Port**. By default, **22** is reserved in TrueNAS.
+   If selecting **SSH private key stored in user's home directory**, the public key for the SSH connection must be saved in the home directory for administration user.
```

Comment:
Try to rephrase to remove possessive here if possible

Reply:
This is what it says in the UI, which is the only time this is allowed...until the UI team changes this.